deckhouse-controller/pkg/controller/deckhouse-release/controller.go

```diff
@@ -633,19 +642,21 @@ func (r *deckhouseReleaseReconciler) runReleaseDeploy(ctx context.Context, dr *v
 
 var ErrDeploymentContainerIsNotFound = errors.New("deployment container is not found")
 
-func (r *deckhouseReleaseReconciler) bumpDeckhouseDeployment(ctx context.Context, dr *v1alpha1.DeckhouseRelease) error {
+func (r *deckhouseReleaseReconciler) bumpDeckhouseDeployment(ctx context.Context, dr *v1alpha1.DeckhouseRelease) (bool /* dryrun? */, error) {
+	var dryrun bool
 	key := client.ObjectKey{Namespace: deckhouseNamespace, Name: deckhouseDeployment}
 
 	depl := new(appsv1.Deployment)
 
 	err := r.client.Get(ctx, key, depl)
 	if err != nil {
-		return fmt.Errorf("get deployment %s: %w", key, err)
+		return dryrun, fmt.Errorf("get deployment %s: %w", key, err)
```

Comment:
if we make return values like this - it looks more clear.
(because we do not need to calculate function result)
```suggestion
		return false, fmt.Errorf("get deployment %s: %w", key, err)
```